pkgs/top-level/all-packages.nix

```diff
@@ -9323,7 +9329,7 @@ with pkgs;
   icingaweb2-ipl = callPackage ../servers/icingaweb2/ipl.nix { };
   icingaweb2-thirdparty = callPackage ../servers/icingaweb2/thirdparty.nix { };
   icingaweb2 = callPackage ../servers/icingaweb2 { };
-  icingaweb2Modules = {
+  icingaweb2Modules = lib.recurseIntoAttrs {
```

Comment:
In all-packages, we have `recurseIntoAttrs` in scope and don't need all the `lib.` prefixes here.